react.js

```diff
@@ -0,0 +1,94 @@
+
+var IntroSection = function (_React$Component) {
+  _inherits(IntroSection, _React$Component);
+
+  function IntroSection(props) {
+    _classCallCheck(this, IntroSection);
+
+    return _possibleConstructorReturn(this, (IntroSection.__proto__ || Object.getPrototypeOf(IntroSection)).call(this, props));
+  }
+
+  _createClass(IntroSection, [{
+    key: "render",
+    value: function render() {
+
+      return this.props.data.map(function (item) {
+        return React.createElement(
+          "section",
+          { className: "intro " + item.style },
+          React.createElement(
+            "div",
+            { className: "intro-text" },
+            React.createElement(
+              "h2",
+              null,
+              item.name
+            ),
+            React.createElement(
+              "p",
+              null,
+              item.content
+            ),
+            React.createElement(
+              "div",
+              { className: "btn" },
+              "Learn More"
+            )
+          ),
+          React.createElement(
+            "div",
+            { className: "intro-img" },
+            React.createElement("img", { className: "desktop", src: item.desktopImgSrc, alt: "" }),
+            React.createElement("img", { className: "mobile", src: item.mobileImgSrc, alt: "" })
+          )
+        );
+      });
+    }
+  }]);
+
+  return IntroSection;
+}(React.Component);
+
+var App = function (_React$Component2) {
+  _inherits(App, _React$Component2);
+
+  function App(props) {
+    _classCallCheck(this, App);
+
+    var _this2 = _possibleConstructorReturn(this, (App.__proto__ || Object.getPrototypeOf(App)).call(this, props));
+
+    _this2.state = [{
+      name: 'Smith & Jones Architects',
+      content: 'Et sed autem causae appareat, tempor abhorreant te mei, facer facilisis sit ea. Eu timeam vidisse consectetuer sed. Duo etiam laboramus dissentiet in, nec no errem',
+      desktopImgSrc: 'img/home/home-img-1.png',
+      mobileImgSrc: 'img/home/home-mobile-img-1.png',
+      style: null
+    }, {
+      name: 'Futuristic Designs',
+      content: 'Et sed autem causae appareat, tempor abhorreant te mei, facer facilisis sit ea. Eu timeam vidisse consectetuer sed. Duo etiam laboramus dissentiet in, nec no errem',
+      desktopImgSrc: 'img/home/home-img-2.png',
+      mobileImgSrc: 'img/home/home-mobile-img-2.png',
+      style: 'designs'
+    }];
+
+    return _this2;
+  }
+
+  _createClass(App, [{
+    key: "render",
+    value: function render() {
+      return React.createElement(IntroSection, { data: this.state });
+    }
+  }]);
+
+  return App;
+}(React.Component);
+
```

Comment:
i am not familiar with the syntax of this type of react.js file, very interesting